recipes/finemap/meta.yaml

```diff
@@ -1,37 +1,44 @@
-# This file created by conda-build 3.22.0
-# meta.yaml template originally from:
-# /home/conda/conda-forge/recipes/finemap, last modified Thu Oct 20 22:53:08 2022
-# ------------------------------------------------
+{% set name = "finemap" %}
+{% set version = "1.4.2" %}
 
 package:
-  name: finemap
-  version: 1.4.1
+  name: {{ name }}
+  version: {{ version }}
+
 source:
-  sha256: 75c919d9cc981bc08e1a982b1632110109ffe6a057e803c9aa36c143aa56d546
-  url: http://www.christianbenner.com/finemap_v1.4.1_x86_64.tgz
+  url: http://www.christianbenner.com/finemap_v{{ version }}_x86_64.tgz # [linux]
+  sha256: 3b1fc6eb3c2ccafd647b32e02d0244495cd0ade9ed7d474606c31ebf6e98b0c9 # [linux]
+  url: http://www.christianbenner.com/finemap_v{{ version }}_MacOSX.tgz # [osx]
+  sha256: e2967e5586347a8ac5cbcddf3c0e41bf8d3e7793875f78b509d07903d17a415b # [osx]
+
 build:
-  binary_relocation: false
-  noarch: generic
-  number: '0'
+  number: 0
+  # skip broken osx for now
+  skip: True # [not linux]
+  run_exports:
+    - {{ pin_subpackage('finemap', max_pin="x") }}
   script:
-    - install -Dd ${PREFIX}/bin
-    - install -D -m 755 finemap_v1.4.1_x86_64 ${PREFIX}/bin/finemap
-  string: '0'
+    - mkdir -p ${PREFIX}/bin
+    - install -m 755 finemap_v{{ version }}_x86_64 ${PREFIX}/bin/finemap # [linux]
+    - install -m 755 finemap_v{{ version }}_MacOSX ${PREFIX}/bin/finemap # [osx]
+
 requirements:
-  build: []
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
Is the compiler needed if there is no build ? 
Above I see a script section that just copies the binaries to PREFIX/bin

Reply:
I added the compiler here to get around the linter, which will otherwise complain that the package should be marked as noarch. Here's the error message I'm getting now
```
ERROR: recipes/finemap/meta.yaml:23: should_be_noarch_generic: The recipe should be build as `noarch`
```

Do you know if there's a better solution? I had borrowed this trick from the plink2 recipe:
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/plink2/meta.yaml

Reply:
IMO you should add a skip for the rules

Reply:
ok, done! I removed the compiler requirement and added a few more dependencies that it seems like the compiler requirement must have been bringing in

I also had to add a few lint skips